content/glossary/endpoint.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: endpoint
+synonyms:
+    - endpoints
+    - method
+    - methods
+    - service
+    - services
+excerpt: A single callable service within an API.
+domains:
+    - software-engineering
+---
+When discussing [APIs](/glossary/application-programming-interface-api), an "endpoint" is a single service that is offered by a [host](/glossary/host) software that can be requested by a [client](/glossary/client). An endpoint on an API hosted on the internet is usually represented by a URL, such as `https://api.kittycad.io/user`.
+
+Some endpoints may ask the host fetch and return some data, like [getting your user](https://kittycad.io/docs/api/get-your-user), or ask them to perform some work for the client and return the result, like [converting a CAD file](https://kittycad.io/docs/api/convert-cad-file).
```

Comment:
`Some endpoints may ask the host fetch and return some data` I feel like something's going awry here, is it meant to be "ask the host **to** fetch and"